girder/cli.py

```diff
@@ -0,0 +1,14 @@
+from click_plugins import with_plugins
+from pkg_resources import iter_entry_points
+import click
+
+
+@with_plugins(iter_entry_points('girder.cli_plugins'))
+@click.group()
```

Comment:
Want to give this a `short_help`? If you can't think of anything better, you could always recycle `'Girder: data management platform for the web.'`.